server/pulp/server/managers/auth/cert/cert_generator.py

```diff
@@ -161,42 +161,19 @@ def decode_admin_user(self, encoded_string):
 
 class SerialNumber:
 
-    PATH = config.config.get('security', 'serial_number_path')
-    __mutex = RLock()
     __metaclass__ = Singleton
 
-    def next(self):
+    def getSerialNumber(self):
         """
-        Get the next serial#
-        @return: The next serial#
+        Returns a uuid to use for the next serial number. x509 standards indicate that serial
```

Comment:
Since this is a multiline docstrings, can a 1-line summary line be added? PEP 257 talks about the multiline format [here](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).